nixos/modules/system/boot/systemd.nix

```diff
@@ -136,6 +139,16 @@ let
       "systemd-ask-password-wall.path"
       "systemd-ask-password-wall.service"
 
+      # Varlink APIs
+      "systemd-bootctl@.service"
+      "systemd-bootctl.socket"
+      "systemd-creds@.service"
+      "systemd-creds.socket"
```

Comment:
Source is https://raw.githubusercontent.com/systemd/systemd/main/units/systemd-creds.socket.